allennlp_models/modelcards/vqa-vilbert.json

```diff
@@ -38,21 +38,24 @@
         "dataset": {
             "name": "VQA dataset",
             "url": "https://visualqa.org/",
-            "notes": "Evaluation requires a large amount of images to be accessible locally, so we cannot provide a command you can easily copy and paste."
+            "notes": "Evaluation requires a large amount of images to be accessible locally, so we cannot provide a command you can easily copy and paste. The first time you run it, you will get an error message that tells you how to get the rest of the data.",
+            "processed_url": "balanced_real_val"
```

Comment:
So, the training_config will interpret `balanced_real_val` correctly?

Reply:
Yes. This is a dataset reader that doesn't take files. It takes just a string instead.